Sources/ApiVideoPlayer/Models/NowPlayingData.swift

```diff
@@ -0,0 +1,18 @@
+import CoreMedia
+import Foundation
+
+struct NowPlayingData {
+    let duration: CMTime
+    let currentTime: CMTime
+    let isLive: Bool
+    let thumbnailUrl: String?
+    let title: String?
+
+    init(duration: CMTime, currentTime: CMTime, isLive: Bool, thumbnailUrl: String?, title: String? = nil) {
```

Comment:
Is this just to initialize `title` to `nil`?
Could you just decalre title like this:
```
let title: String? = nil
```